pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -286,10 +286,10 @@ const PageSystemInformationChangeHostname = () => {
         >
             <Form isHorizontal onSubmit={onSubmit}>
                 <FormGroup fieldId="sich-pretty-hostname" label={_("Pretty host name")}>
-                    <TextInput id="sich-pretty-hostname" value={pretty} onChange={onPrettyChanged} />
+                    <TextInput id="sich-pretty-hostname" value={pretty} onChange={(_event, value) => onPrettyChanged(value)} />
                 </FormGroup>
                 <FormGroup fieldId="sich-hostname" label={_("Real host name")}>
-                    <TextInput id="sich-hostname" value={hostname} onChange={onHostnameChanged} validated={error.length ? "error" : "default"} />
+                    <TextInput id="sich-hostname" value={hostname} onChange={(_event, value) => onHostnameChanged(value)} validated={error.length ? "error" : "default"} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-121108-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)